selfdrive/car/mazda/carstate.py

```diff
@@ -72,6 +72,8 @@ def update(self, cp, cp_cam):
         self.lkas_allowed_speed = True
       elif speed_kph < LKAS_LIMITS.DISABLE_SPEED:
         self.lkas_allowed_speed = False
+    else:
+      self.lkas_allowed_speed = True
```

Comment:
why not just set this to true in the constructor?

Reply:
On cars where minSteerSpeed isn't 0, then behavior will be different if openpilot starts up and we're in between the enable speeds. Before it would default to False which would be correct, but now it would be True, incorrectly allowing lkas when we're still under ENABLE_SPEED